kibana_prometheus_exporter/config.py

```diff
@@ -45,6 +48,15 @@ def description(self):
             config_list.append(("Kibana login:", self.kibana_login))
             config_list.append(("Kibana password:", "***"))
 
+        if self.ignore_ssl:
+            config_list.append(("SSL verification:", "disabled"))
+        else:
+            config_list.append(("SSL verification:", "enabled"))
+            if self.requests_ca_bundle and os.path.isfile(self.requests_ca_bundle):
```

Comment:
I think this isn't the right place to check for valid configuration.
If the check is to be done by this module, it should be done in a `check_bundle` function as the other checks.
The purpose of `description` is to print whatever the configuration is.